### PR TITLE
Adding  Product Listing Page.

### PR DESCRIPTION
**Pull request by manojkumar8400**
Mapped product data from DB and added some filter features
Screenshot from 2022-04-01 00-49-29
.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,10 +5,11 @@ import { Navigation, Footer } from './components/index.js';
 import { Home, AllProducts, Login, Signup, MyCart, SingleProduct, WishList } from "./pages/index.js";
 
 function App() {
-  console.log("bb")
+  console.log("dsd")
```
Comment (bipin7yadav) [inline src/App.js:8]: remove unnecessary comments

File: src/components/ProductList/ProductList.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
+.product-container {
+    display: grid;
+    grid-template-columns: 13rem 1fr;
+    grid-gap: 15px;
```
Comment (bipin7yadav) [inline src/components/ProductList/ProductList.css:4]: you can use rem instead of pixels

**Review by bipin7yadav — APPROVED** [review]

**Review by gopalbbharadva — APPROVED** [review]
All looks working , check once why delpy preview of netlify not working?

**Comment by manojkumar8400** [comment]
> All looks working , check once why delpy preview of netlify not working?

Ok, I"ll check.

**Pull request merged**